instrumentation/slowrendering/src/main/java/io/opentelemetry/android/instrumentation/slowrendering/SlowRenderListener.java

```diff
@@ -196,6 +196,7 @@ private void reportSlow(PerActivityListener listener) {
     }
 
     private void makeSpan(String spanName, String activityName, int slowCount, Instant now) {
+        // TODO: Use an event rather than a zero-duration span
```

Comment:
should we create an issue for this?

Reply:
https://github.com/open-telemetry/opentelemetry-android/issues/904